src/System.Private.CoreLib/shared/System/IO/Path.cs

```diff
@@ -45,7 +46,7 @@ public static partial class Path
         // returns null. If path does not contain a file extension,
         // the new file extension is appended to the path. If extension
         // is null, any existing extension is removed from path.
-        public static string ChangeExtension(string path, string extension)
+        public static string? ChangeExtension(string? path, string? extension) // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/26761
```

Comment:
Is the `TODO-NULLABLE` comment referring that we should annotate this somehow that it will only return null if the input is null?

Reply:
Yes